product/interfaces/controller/apicontroller.go

```diff
@@ -19,15 +20,15 @@ type (
 
 	// APIResult view data
 	APIResult struct {
-		Error   *resultError
+		Error   *productResultError
```

Comment:
do we need this rename? guess by reverting the swaggo version we can keep the file unchanged..

Reply:
returned